solver/pb/ops.proto

```diff
@@ -45,6 +45,7 @@ message ExecOp {
 	repeated Mount mounts = 2;
 	NetMode network = 3;
 	SecurityMode security = 4;
+	repeated Op dependencies = 5;
```

Comment:
This should be `repeated ExecOp dependencies` + some extra fields for the port connections in the future. Inputs are shared with the main inputs array. We probably would like to extend `Input` definition with `bool remoteAllowed` (or enum) to signify inputs that can remain in separate nodes but probably bit early for that atm. In the end, there should also be a way to mark helper processes that can be shared by multiple vertexes. That should change the proto definition much though, just need more fields to mark unique processes and shutdown routine.

Reply:
Thanks for your feedback. We'll take into account your suggestions.

Yes, at first sight I tried to use `Op` directly to have auto-contained information in `dependencies` attribute. But, afterwards I used the `SourceOp` info from the root `Op` to take advantage of the `Source` resolver.